playwright-e2e/dsm/pages/samples/report-page.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, Page } from '@playwright/test';
+import DatePicker from 'dsm/component/date-picker';
+
+export default class ReportPage {
+  constructor(private readonly page: Page) {}
+
+  async waitForReady(): Promise<void> {
+    await expect(this.page.locator('h1')).toHaveText('Report');
+  }
+
+  async pickStartDate(opts: { yyyy?: number, month?: number, dayOfMonth?: number } = {}): Promise<string> {
+    const { yyyy, month, dayOfMonth } = opts;
```

Comment:
just to be sure - since they're already marked as optional when listed as parameters, what does this line do?

Reply:
Assigns default values if param is undefined or null.